towhee/dag/operator_repr.py

```diff
@@ -30,26 +29,53 @@ class OperatorRepr(BaseRepr):
         df_out:
             This operator's output dataframe.
     """
-
-    def __init__(self, name: str, df_in: DataframeRepr, df_out: DataframeRepr):
+    def __init__(
+        self,
+        name: str,
+        dataframes: Dict[str, DataframeRepr],
+        src: str = None,
+    ):
         super().__init__(name)
-        self._df_in = df_in
-        self._df_out = df_out
-        self._iter_in = None
-        self._iter_out = None
+        self._inputs = {}
+        self._outputs = {}
+        if src:
+            self.from_yaml(src, dataframes)
 
     @property
-    def df_in(self):
-        return self._df_in
+    def inputs(self):
+        return self._inputs
 
     @property
-    def df_out(self):
-        return self._df_out
+    def outputs(self):
+        return self._outputs
 
-    @df_in.setter
-    def df_in(self, value: DataframeRepr):
+    @inputs.setter
+    def inputs(self, value: Dict[str, DataframeRepr]):
         self._df_in = value
 
-    @df_out.setter
-    def df_out(self, value: DataframeRepr):
+    @outputs.setter
+    def outputs(self, value: Dict[str, DataframeRepr]):
         self._df_out = value
+
+    def from_yaml(self, src: str, dataframes: Dict[str, DataframeRepr]):
```

Comment:
staticmethod, return a repr, can use another function to add  DatafremeRepr object

Reply:
fixed